### ISSUE TITLE
Support for Other Distill Layouts

### ISSUE THREAD

**Issue by saptarshiguha** [rstudio/distill#95]
Hi,
I'm back! Going to commit my self to this format. I notice on the distill website there are several layout options for figures e.g. .l-body.side,.l-middle.side,.l-page.side.

Will these be supported? i tried using them but they aren't (or maybe i did something wrong?)

**Comment by astamm** [comment]
Hi,
I just set up my first Distill website and ended up having the same issue. It seems that floated images is not yet supported by distill despite the documentation that announces the existence of layouts such as `l-body.side` which are currently not recognised though.
I would appreciate any comments on these features.
Thanks.

**Comment by cderv** [comment]
Regarding `l-body.side`, it was answered in https://github.com/rstudio/distill/issues/263#issuecomment-751810989

**Comment by astamm** [comment]
Thanks for pointing out the solution. It seems to be working as expected with `layout = "l-body side"` as chunk option.

**Comment by cderv** [comment]
Yes `.l-body.side` is equivalent to `layout = "l-body side"` but I don't think this give what is still documented in https://distill.pub/guide/ which was working for V1 but not V2.

Seems now that `side` is another way to put on the `aside`: https://rstudio.github.io/distill/#footnotes-and-asides

**Comment by astamm** [comment]
You are right. It seems that `layout = "l-body side"`is interpreted as an `aside` environment now. It is not inserting the image as a floating image inline in the text.

**Comment by cderv** [comment]
Yes this is it. And it seems Distill V2 does not support the inline floating image anymore. That is why the R package **distill** does not have it as we are based on V2.
